definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -73,7 +73,7 @@ declare module 'express' {
     location(path: string): this;
     redirect(url: string, ...args: Array<void>): this;
     redirect(status: number, url: string, ...args: Array<void>): this;
-    render(view: string, locals?: mixed, callback: (err?: ?Error) => mixed): this;
+    render(view: string, locals?: mixed, callback: (err?: ?Error, html: string) => mixed): this;
```

Comment:
Nit: This type is a little odd. Flow probably should be changed to ban non-optional parameters from following optional parameters.

If the callback method is always passed two arguments

1. `err`, which is either an instance of `Error` or `null`
2. `html`, which is a string

then the most correct type would be `(err: Error | null, html: string) => mixed)`.

However, as written, this is basically `(err: Error | null | void, html: string) => mixed)`, which is only a little bit more strict than it needs to be.

So I'm happy to merge without this being fixed. Just let me know what you want to do!